packages/atomic/src/components/search/atomic-refine-toggle/atomic-refine-toggle.new.stories.tsx

```diff
@@ -40,3 +40,56 @@ export const Default: Story = {
     `,
   ],
 };
+
+export const WithAtomicExternals: Story = {
+  name: 'atomic-refine-toggle',
+  decorators: [
+    (story) => html`
+      <atomic-search-interface id="foo" data-testid="root-interface">
```

Comment:
It's purposefully mixed up

Reply:
Maybe add a comment to ensure that no one accidentally boy-scouts it clean 😅

Reply:
They can try, the tests will bite them 😈